src/main/java/com/dakar/dakar/validator/GraphQLValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.dakar.dakar.validator;
+
+import com.coxautodev.graphql.tools.SchemaParser;
+import com.dakar.dakar.models.GraphQLParameter;
+import com.dakar.dakar.resolvers.JourneyResolver;
+import com.dakar.dakar.resolvers.MutationResolver;
+import com.dakar.dakar.resolvers.QueryResolver;
+import com.dakar.dakar.services.interfaces.IJourneyService;
+import graphql.language.Document;
+import graphql.parser.Parser;
+import graphql.schema.GraphQLSchema;
+import graphql.validation.ValidationError;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class GraphQLValidator {
+
+    @Autowired
+    private IJourneyService journeyService;
+
+    public List<ValidationError> validateGraphQL(GraphQLParameter graphQLParameter){
+        GraphQLSchema graphQLSchema = SchemaParser.newParser()
```

Comment:
i think we already have a graphqlschema defined at the boot time of our spring app, please see main class